raven/src/main/java/org/opencadc/raven/RavenInitAction.java

```diff
@@ -173,6 +192,38 @@ private void initKeys() {
         log.info("initKeys: OK");
     }
 
+    private void initAvailabilityCheck() {
+        StorageSiteDAO storageSiteDAO = new StorageSiteDAO();
+        storageSiteDAO.setConfig(getDaoConfig(props));
+
+        this.siteAvailabilitiesKey = this.appName + RavenInitAction.JNDI_AVAILABILITY_NAME;
+        terminate();
+        this.availabilityCheck = new Thread(new AvailabilityCheck(storageSiteDAO, this.siteAvailabilitiesKey));
+        this.availabilityCheck.setDaemon(true);
+        this.availabilityCheck.start();
+    }
+
```

Comment:
How does terminate() get called? 

I don't see a RavenInitAction.doShutdown() implementation anywhere...

Reply:
Yeah if you're going to unbind anyways, might as well just do the unbind. Updated.

Oops, missed the doShutdown(), added.